### ISSUE TITLE
Find a new venue for the pub meet

### ISSUE THREAD

**Issue by henare** [openaustralia/oaf#211]
At July's pub meet we want to have lightning talks with projector available. While I think the Woolpack can provide this I've had a number of people point out problems with the venue so it seems like a good opportunity to find a new place.

It was really hard for me to decide on the Woolpack because I considered a number of factors. For our next venue we're going to have to consider even more factors so it will not be an easy job. Hence this issue is to outline our requirements and start to gather suggestions.

### Requirements

* Good food
* Good beer
* Room/space available for at least 25 people with projector
* Within 10 minutes walking distance from Central
* Safe walk to Central

#### [Woolpack Hotel](http://woolpack.com.au/)
Good:
* Within 10 minutes walking distance from Central
* Decent beer selection

Bad:
* Dark walk to Central late at night
* Food's pretty average. Limited vegetarian options
* Staff can be pretty random and unhelpful
* Projector not available (there may be one in the restaurant but that's not a good space for us)

**Comment by henare** [comment]
What else have I missed? Please comment or edit the summary directly :green_heart: 

Ping @mlandauer @equivalentideas @kat @forkinthecode. Please @mention anyone else you can thing would like to participate.

**Comment by henare** [comment]
A couple of ideas/suggestions from last night:

* King St Brewhouse
* Trinity Bar

**Comment by henare** [comment]
Jason suggested the [Metropolitan Hotel](http://metropolitanhotelsydney.com.au/). I've edited a few of his comments:

>Equidistant from both Wynard and Circular Quay. Very easy to get to and very safe.
>...they have good pub food, good selection of beers, all reasonably priced, and a great little rooftop terrace for drinks before hand.
>...a private dining room could be dedicated for our meetings, which we can use WITH a projector AND data if we need it. And if we get bigger groups, there are several other areas that we can migrate to with full PA and mics etc.

**Comment by forkinthecode** [comment]
I'm also searching for a venue in the city as I've noticed people don't like to go out to Marrickville so I've been asking around. 

The Sydney Python meetup meets at the Hotel Sweeny http://www.meetup.com/sydneypython/events/221908407/ and I've been told there is a room there with a projector that may be able to be booked for free although the bad point is that there can be noisy things going on upstairs. It does not sound like a terribly fashionable place but I'm yet to see it personally.

I'm making enquiries at the library room at the Crown Hotel which seats only 16 (mainly for my discussion group) as there is no wifi etc and am waiting on a reply. 

The Metropolitan sounds very nice. I was concerned with the nicer pubs I looked at that they would not allow groups to book for free but I guess that may depend on the pub.

If you don't want to use the Woolpack maybe I'll use it instead. I do kind of like that it has that area that can be booked, I just thought if I made my discussion group there that people would get too confused between the two groups. Also wanted to check out other pubs for the fun of it though ;-)
